tools/system_libs.py

```diff
@@ -1295,6 +1297,11 @@ def retrieve():
           f = urlopen(url)
           data = f.read()
 
+      if sha512hash:
+        actual_hash = hashlib.sha512(data).hexdigest()
+        if actual_hash != sha512hash:
+          raise RuntimeError('Unexpected hash: ' + actual_hash + '\n'
+                             'If you are updating the port, please update the hash in the port module.')
```

Comment:
how about putting the `# To compute the sha512 hash, run `curl URL | sha512sum` comment in parentheses in this error message? otherwise I'm not sure people will see the source comment above, but they will definitely see this error.

Reply:
Alternatively we could just delete that comment since the error message already contains the new computed hash.